CONTRIBUTING.rst

```diff
@@ -194,6 +194,27 @@ title                                  string
 
 For a full schema of a category JSON object, please see https://github.com/pyvideo/data/blob/master/.schemas/category.json
 
+Creating Data From Youtube Channel
+~~~~~~~~~~~~
+
+You can easily create basic data from already created youtube channel
```

Comment:
Hi, dropping by to provide editorial two cents as per @darjeeling's request. I am not sure what "basic data" means here. "easily" is also (usually!) unnecessary. As I understand, the point here is to automatically extract the JSON file from a youtube list, so this should be stated explicitly. How about something like:
- "You can extract the JSON data from the pyvideo YouTube list."